docs/pages/config-reference.mdx

```diff
@@ -9,6 +9,9 @@ Teleport uses the YAML file format for configuration. A full configuration refer
 file is shown below, this provides comments and all available options for `teleport.yaml`
 By default, it is stored in `/etc/teleport.yaml`.
 
+This document aims to be a reference rather than a starting point for a real cluster. To
+get a good starting file, run `teleport configure > teleport.yaml`.
```

Comment:
This document is a reference rather than a starting point for a real cluster.
 To get a good starting file, run `teleport configure -o file`.